doc/users/explain/writing_a_backend.rst

```diff
@@ -0,0 +1,79 @@
+=================
+Writing a backend
+=================
+
+This page assumes general understanding of the information in the
+:doc:`/users/explain/backends` page, and is instead intended as reference for
+third-party backend implementers.  It also only deals with the interaction
+between backends and `.pyplot`, not with the rendering side, which is described
+in `.backend_template`.
```

Comment:
This split is somewhat inconvenient. Ok for a start, but in the long term, "writing a backend" should be summarized in one place. You could call this page "Interaction between backends and pyplot" and thus shift its focus.